.github/CODEOWNERS

```
@@ -1,2 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
+
+*                                    @MetaMask/shared-libraries-engineers
```

The shared libraries team is mainly involved in the code, not the config. Perhaps we can just list the team as codeowners of the code itself, along with the manifest, lockfile, and actions. That ensures we have full ownership of the code, packaging, and publishing, but leaves the config file untouched.

```suggestion
*.js                                 @MetaMask/shared-libraries-engineers
package.json               @MetaMask/shared-libraries-engineers
.github/                        @MetaMask/shared-libraries-engineers

```